cypress/integration/ete/email/template.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Email template generation', () => {
```

Comment:
just noticed that some more email tpls were added to the switch-case below, do they need a test in-place as well?

Reply:
Haha, d'oh!